src/utils/FeatureFinderMetaboIdent.cpp

```diff
@@ -261,11 +261,11 @@ class TOPPFeatureFinderMetaboIdent :
     mzml.setLogType(log_type_);
     mzml.getOptions().addMSLevel(1);
     mzml.load(in, ff_mident.getMSData());
-    if (ff_mident.getMSData().empty() && !force)
+    if (ff_mident.getMSData().empty())
     {
-      OPENMS_LOG_ERROR << "Error: No MS1 scans in '"
-                       << in << "' - aborting." << endl;
-      return INCOMPATIBLE_INPUT_DATA;
+      OPENMS_LOG_WARNING << "Error: No MS1 scans in '"
+                         << in << "' - aborting." << endl;
+      if(!force) return INCOMPATIBLE_INPUT_DATA;
```

Comment:
So now the problem is that we get the message twice in the force case, and its a warn rather than an error.

Reply:
you are right.. I will undo this